test/src/module/command/helpTest.c

```diff
@@ -124,6 +124,8 @@ testRun(void)
             "\n"
             "Command Options:\n"
             "\n"
+            "  --archive-mode                   disable archiving on restored cluster\n"
```

Comment:
Seems like this would be better as "preserve or disable archiving" or something along those lines..  seems a bit odd to describe it only as "disable"

Reply:
Preserve is the default -- so if you are using it you are probably disabling. But I'm fine with the new language as well.